validation/markets-validator.js

```diff
@@ -464,7 +481,48 @@ function MetadataValidator(metadata) {
             this.errors.push(new NullAttributeIssue("url"));
         } else if (url.length === 0) {
             this.errors.push(new EmptyAttributeIssue("url"));
-        }
+        } else {
+            this.validateUrlStatus(url);
+        }
+    };
+
+    this.validateUrlStatus = function (url) {
```

Comment:
- Please remove space here `function (url)` for consistency.
- Please use a dedicated _issue_ such as:

``` js
function UnknownProtocolIssue(cityName, protocol) {

    this.cityName = cityName;
    this.protocol = protocol;

    this.toString = function() {
        return this.cityName + "Unknown protocol '" + this.protocol + "' in data source url.";
    };
}
```